### DISCUSSION TITLE
Daily Calorie Counter

### DISCUSSION THREAD

**Discussion by teolemon** [openfoodfacts/openfoodfacts-androidapp#3996] [Ideas]
Example of what could be done
https://material.uplabs.com/posts/calorie-carb-fat-counter-android-app

Also need to be able to input in natural language: "I ate half of this barcode" >> the app scans the barcodes and makes the calculations

![image](https://cloud.githubusercontent.com/assets/1689815/21388897/e5411b80-c77e-11e6-8a5a-f6a9b0fd89ac.png)

**Comment by wdwde** [comment]
I would like to see this feature :+1:

**Comment by gayanW** [comment]
We'd like to work on this. Do we need calcualte the burned amount as in the picture? Will that be really necessary?

Any ideas are welcome.

**Comment by lenucksi** [comment]
A general open-source version of the usual intake counting apps would be great, especially given the OpenFoodFacts data as a basis.
I would like to add [FDDBExtender](https://fddbextender.de) as a nice example, too. They do server storage and viz of the recorded data too and provide an opportunity to collect weight metrics and broad sport activity too. A nice, self-hosted alternative to said features would be great too.

**Comment by alexisdal** [comment]
I need a calorie counter too and I'm unhappy with the few options offered on Play Store. Not only apps are bloated and show useless ads, but most annoyingly the data is private. Besides either product does not exist in database and/or contains wrong food facts.
I thought apps most would use openfoodfacts but no.
And looking into openfoodfacts, i ended up here.

I disagree with the idea to have calorie counter as a feature of today's openfoodfacts android-app. It is an entirely different usecase and it needs its app.

Can someone confirm that there is no such app as of today?
